<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HouseNumbering_update_selected_form</class>
 <widget class="QDialog" name="HouseNumbering_update_selected_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>235</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>235</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>235</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HouseNumbering</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>155</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="CBfields">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>30</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="QLEvalore">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>60</y>
     <width>65</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Set value for the data field:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Start value: (example1 or 1a)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>195</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>You can also activate this form with F10 funct key - by Marco Braida 2014</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cBkeepLatestValue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>155</y>
     <width>251</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Remember latest inserted  value</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="QLEnsteps">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>45</width>
     <height>24</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="cBkeepCharFixed">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>95</y>
     <width>541</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>based on start value keep letter part fixed and increase only numeric part</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>10</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>numeric step value</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cBsetLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>125</y>
     <width>426</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>set the working field as active label for current layer</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>QLEvalore</tabstop>
  <tabstop>CBfields</tabstop>
  <tabstop>cBkeepLatestValue</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HouseNumbering_update_selected_form</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HouseNumbering_update_selected_form</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
